Config.cmake.in

```diff
@@ -14,12 +14,12 @@ if(${CMAKE_VERSION} VERSION_LESS "3.15.0")
   find_package(OpenCV QUIET)
 else()
   # we do not add the other dependencies because these are header files lib
-  find_dependency(mpi_cmake_modules REQUIRED)
-  find_dependency(cereal REQUIRED)
-  find_dependency(Boost REQUIRED COMPONENTS iostreams)
+  find_dependency(mpi_cmake_modules)
+  find_dependency(cereal)
+  find_dependency(Boost COMPONENTS iostreams)
 
   # optional dep
-  find_dependency(OpenCV QUIET)
+  find_package(OpenCV QUIET)
```

Comment:
for my education: what is the difference ?

Reply:
It forwards the `REQUIRED` or `QUIET` parameters of the original `find_package` call (i.e. `find_package(serialization_utils ...)`).  I think the reasoning is that if `serialization_utils` itself is only required as optional dependency, it should not fail the build if one of its dependencies is not there.
On the other hand, this means that if `serialization_utils` is required, all the `find_dependency` calls automatically become required too (to my understanding the `QUIET` parameter was simply ignored here), which is not desired for optional dependencies.
See https://stackoverflow.com/a/64847834/2095383